clubs/views.py

```diff
@@ -112,13 +114,21 @@ def children(self, request, *args, **kwargs):
         return Response(child_tree)
 
     @action(detail=True, methods=['get'], url_path='notes-about')
-    def notes_about(self, request, *args, **kwards):
+    def notes_about(self, request, *args, **kwargs):
         club = self.get_object()
         queryset = Note.objects.filter(subject_club__code=club.code)
         queryset = filter_note_permission(queryset, club, self.request.user)
         serializer = NoteSerializer(queryset, many=True)
         return Response(serializer.data)
 
+    @action(detail=True, methods=['get'])
+    def qr(self, request, *args, **kwargs):
+        url = 'https://pennclubs.com/club/{}'.format(self.kwargs['code'])
```

Comment:
why not use an fstring here?

Reply:
I just looked that up, I've never used fstring before it's so clean

Reply:
ikr. super pretty